### DISCUSSION TITLE
gmail Pi.alert en doker

### DISCUSSION THREAD

**Discussion by arkaild** [netalertx/NetAlertX#204] [General]
the configuration with gmail or yahoo is not working. I don't know what is the problem. I have tested with SSL, not SSL, TSL, 487 or 465. 

   [API] Updating table_custom_endpoint.json file in /front/api
    Changes detected, sending reports
      Sending report by Email
[LOG_LEVEL=debug] 20:12:33 REPORT_TO: j*******c@gmail.com  SMTP_USER: j*******c@gmail.com
[LOG_LEVEL=debug] 20:12:33 SMTP try
[LOG_LEVEL=debug] 20:12:33 Trying to open connection to smtp.gmail.com:587
[LOG_LEVEL=debug] 20:12:33 SMTP_FORCE_SSL == True so using .SMTP_SSL()
[LOG_LEVEL=debug] 20:12:33 SMTP_PORT == 0 so sending .SMTP_SSL(SMTP_SERVER, SMTP_PORT)

**Comment by jokob-sk** [comment]
Hi there,

Did you try following these steps?

https://github.com/jokob-sk/Pi.Alert/blob/main/docs/SMTP_GMAIL.md

J
